src/app/pages/Home/partials/ProductItem.tsx

```diff
@@ -1,55 +1,74 @@
-import React from 'react';
-import { IProduct } from './../../../shared/interfaces/product';
-import Badge from './../../../shared/components/partials/Badge';
-import Button from './../../../shared/components/partials/Button';
+import React from "react";
+import { Link } from "react-router-dom";
+import { useGlobalContext } from "../../../shared/contexts/cart.context";
+import { IProduct } from "./../../../shared/interfaces/product";
+import { ICart } from "./../../../shared/interfaces/cart";
```

Comment:
Rank B: Thực chất ở cart cũng chỉ là 1 product trong giỏ hàng, nên tạo interface tách product với cart ra là sai hoàn toàn nghen